alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -392,11 +393,15 @@ private function getAutoEmbedCode($targetId = null)
 		}
 
 		$entry_id = $this->getRequestParameter(self::ENTRY_ID_PARAM_NAME);
-		if (!$entry_id)
-		{
-			KExternalErrors::dieError(KExternalErrors::MISSING_PARAMETER, "Entry ID not defined");
+        $playlist_id = $this->getRequestParameter(self::PLAYLIST_ID_PARAM_NAME);
+        $loadContentMethod = "";
```

Comment:
can u please check indentation @Yuvalke

Reply:
It looks good on my IDE, I'll try another one just in case

Reply:
webstorm: view->Active Editor->Show Whitespaces
![image](https://user-images.githubusercontent.com/5461862/93014006-f6fec980-f5b5-11ea-8952-5227fd854a58.png)